citi-onboarding/maracatronics
# Feature/contact section

## What i did
- The contact section

## How to test
1. First, you need to use the command ```git fetch --all```
2. Change your branch to "feature/contact-section" with ```git checkout feature/contact-section```
3. Now, you need to open two terminals
4. In some terminal, go to client folder, with ```cd client```
5. In the other terminal, go to server folder with ```cd server```
6. In each terminal, run it with ```npm/yarn start``` in the client folder and ```nodemon index.js``` in the server.
7. Now, you need to open the http://localhost:3000/ (the client side)
8. The first part: try to send a message with the forms, if it's work, some message will show in console like "mensagem enviada", and, of course, i'll see that message, so i can send a print to you.
9. The second part: check the carousel, add a new collaborators in keystone and check if it'll show in carousel.
10. The third part: change the social media links in keystone and check if it really changed. (each social media is a link)
